modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/typeclasses/Concurrent.kt

```diff
@@ -222,15 +222,15 @@ interface Concurrent<F> : Async<F> {
    * }
    * ```
    *
-   * @param ctx [CoroutineContext] to execute the source [F] on.
+   * @param this@raceTriple [CoroutineContext] to execute the source [F] on.
    * @param fa task to participate in the race
    * @param fb task to participate in the race
    * @param fc task to participate in the race
    * @return [RaceTriple]
    *
    * @see [arrow.effects.typeclasses.Concurrent.raceN] for a simpler version that cancels losers.
    */
-  fun <A, B, C> raceTriple(ctx: CoroutineContext = dispatchers().default(), fa: Kind<F, A>, fb: Kind<F, B>, fc: Kind<F, C>): Kind<F, RaceTriple<F, A, B, C>>
+  fun <A, B, C> CoroutineContext.raceTriple(fa: Kind<F, A>, fb: Kind<F, B>, fc: Kind<F, C>): Kind<F, RaceTriple<F, A, B, C>>
```

Comment:
Are we not doing `dispatchers().default().raceTriple` ?

Reply:
We want `NonBlocking.raceTriple` or similar as receiver because I think we can make suspend side effects denote whether they perform blocking I/O or not and prevent compilation from happening when you mix the wrong effect types with the wrong CoroutinesContext.